src/main/java/subway/common/exception/CustomException.java

```diff
@@ -0,0 +1,18 @@
+package subway.common.exception;
+
+public class CustomException extends RuntimeException {
```

Comment:
각 에외에 맞는 Exception을 각각 정의해도 좋아요.

```suggestion
public class NotFoundException extends RuntimeException {
```